files/grub.cfg

```diff
@@ -3,33 +3,9 @@
 # file by default, this can be enforced via --foreman-proxy-tftp-replace-grub2-cfg
 # foreman-installer option or Puppet parameter.
 
-insmod regexp
-
-# On Debian/Ubuntu grub2 does not have patch for loading MAC-based configs. Also due to bug
-# in RHEL 7.4 files are loaded with an extra ":" character at the end. This workarounds both
-# cases, make sure "regexp.mod" file is present on the TFTP. For more info see:
-# https://bugzilla.redhat.com/show_bug.cgi?id=1370642#c70
-regexp --set=1:m1 --set=2:m2 --set=3:m3 --set=4:m4 --set=5:m5 --set=6:m6 '^([0-9a-f]{1,2})\:([0-9a-f]{1,2})\:([0-9a-f]{1,2})\:([0-9a-f]{1,2})\:([0-9a-f]{1,2})\:([0-9a-f]{1,2})' "$net_default_mac"
-mac=${m1}-${m2}-${m3}-${m4}-${m5}-${m6}
-configfile=/grub2/grub.cfg-01-$mac
-source "$configfile"
-
-# If MAC-specific config is not found, attempt to boot from local drive.
-
-default=local_chain_hd0
-timeout=20
-
-menuentry 'Chainload into BIOS bootloader on first disk' --id local_chain_hd0 {
-  set root=(hd0,0)
-  chainloader +1
-}
-
-menuentry 'Chainload into BIOS bootloader on second disk' --id local_chain_hd1 {
-  set root=(hd1,0)
-  chainloader +1
-}
-
-menuentry 'Third disk - for EFI or Discovery click on Build PXE Default'' --id local_chain_hd2 {
-  set root=(hd2,0)
-  chainloader +1
-}
+echo "This system was not recognized by Foreman."
+echo "Click on 'Build PXE Default' in the Foreman interface"
+echo "to create the default global grub.cfg configuration."
+echo "This system will poweroff shortly."
+sleep 900
+poweroff
```

Comment:
Looks like a very extreme response to me, especially given the fact that this can concern remote h/w hosts.

Reply:
Suggest something better that prevents confused users when this template does not work. Problem is that by default we try to chainload EFI from ESP here and this has been broken multiple times already. I change it way too often and forget it's here.

Reply:
What's happening without the poweroff?

Reply:
That depends on the firmware I guess, but for libvirt it's grub shell. I can remove that poweroff if you want, I just wanted to prevent endless loops.